modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/lizard.dm

```diff
@@ -5,7 +5,7 @@
 	limbs_icon = 'modular_skyrat/master_files/icons/mob/species/lizard_parts_greyscale.dmi'
 	cultures = list(CULTURES_EXOTIC, CULTURES_LIZARD, CULTURES_HUMAN)
 	learnable_languages = list(/datum/language/common, /datum/language/draconic)
-	payday_modifier = 0.75
+	//SKYRAT EDIT - EQUAL PAY payday_modifier = 0.75
```

Comment:
No need to comment it out here, it's modular.

Reply:
Got it, it's been deleted